src/components/Typography/Typography.stories.tsx

```diff
@@ -1,85 +1,167 @@
-import { useTheme } from '@emotion/react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
 import Highlight from './Highlight';
 
 import Typography from '.';
 
 export default {
-  title: 'Typography',
+  title: 'Design System/Components/Typography',
   component: Typography,
   parameters: {
     layout: 'fullscreen',
+    componentSubtitle:
+      'Typography는 텍스트의 역할에 따라 다양한 폰트 크기와 굵기를 설정할 수 있습니다.',
+  },
+  argTypes: {
+    children: {
+      name: 'children',
+      description: 'Typography로 나타낼 글 내용입니다.',
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    variant: {
+      name: 'variant',
+      description: '텍스트 역할을 선택합니다.',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'body' },
+      },
+      type: { name: 'string', required: true },
+      control: {
+        type: 'select',
+        options: ['title1', 'title2', 'subtitle1', 'subtitle2', 'body', 'desc'],
+      },
+    },
+    color: {
+      name: 'color',
+      description: '텍스트 색상을 지정합니다.',
+      table: {
+        type: { summary: 'string' },
```

Comment:
```suggestion
    docs: {
      description: {
        component:
          '- variant 값으로 `"title1" | "title2" | "subtitle1" | "subtitle2" | "body" | "desc"`를 사용할 수 있습니다.\n' +
          '- 텍스트 부분 하이라이팅을 위해 Highlight 컴포넌트를 함께 사용할 수 있습니다.',
      },
    },
  },
  argTypes: {
    children: {
      name: 'children',
      description:
        'Typography로 나타낼 글 내용입니다. 문자로 평가되는 모든 노드가 올 수 있습니다.',
      table: {
        type: { summary: 'string' },
      },
    },
    variant: {
      name: 'variant',
      description: '텍스트 역할을 선택합니다.',
      table: {
        type: { summary: 'TypographyVariant' },
        defaultValue: { summary: 'body' },
      },
      type: { name: 'string', required: true },
      control: {
        type: 'select',
        options: ['title1', 'title2', 'subtitle1', 'subtitle2', 'body', 'desc'],
      },
    },
    color: {
      name: 'color',
      description: '텍스트 색상을 지정합니다.',
      table: {
        type: { summary: "CSSProperties['color']" },
```

타입 및 추가 설명입니다. 저희가 정의한 Variant는 명시적으로 문서에 적어두는게 좋겠다고 생각했어요.

Reply:
타입 수정하고 `<HightLight>` 컴포넌트에 대한 Props도 추가했습니다.